editoast/src/views/infra/routes.rs

```diff
@@ -36,18 +35,30 @@ enum WaypointType {
     BufferStop,
 }
 
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
+struct WaypointResponse {
+    starting: Vec<String>,
+    ending: Vec<String>,
+}
+
+impl WaypointResponse {
+    pub fn new(starting: Vec<String>, ending: Vec<String>) -> Self {
+        Self { starting, ending }
+    }
+}
+
 /// Return the railjson list of a specific OSRD object
-#[get("/{waypoint_type}/{waypoint}")]
+#[get("/{waypoint_type}/{waypoint_id}")]
```

Comment:
This is not a change you did, but this comment is innacurate and could be confusing.
We should change it.

Reply:
I have changed the `waypoint' by `waypoint_id'

Reply:
Yeah, I did not speak about this changes, but about this comment : 
`/// Return the railjson list of a specific OSRD object`

Reply:
Ok, do you want me to change it? or just remove it? Any suggestions?

Reply:
Better change it.